### PR TITLE
Doc: Add at-id reference tag to Strided Arrays interface section

### PR DESCRIPTION
**Pull request by mbauman**

### CODE REVIEW

File: doc/src/manual/interfaces.md [hunk 1]
```diff
@@ -391,7 +391,7 @@ something other than 1), you should specialize `indices`. You should also specia
 so that the `dims` argument (ordinarily a `Dims` size-tuple) can accept `AbstractUnitRange` objects,
 perhaps range-types `Ind` of your own design. For more information, see [Arrays with custom indices](@ref).
 
-## [Strided Arrays]
+## [Strided Arrays](@id man-interface-strided-arrays)
```
Comment (fredrikekre) [inline doc/src/manual/interfaces.md]: Is this tag used somewhere or what is the reason for this?

Comment (mbauman) [reply doc/src/manual/interfaces.md]: No, I'm just following the convention in this file — and more importantly getting rid of the literal `[]` in the header.

Comment (fredrikekre) [reply doc/src/manual/interfaces.md]: > ... getting rid of the literal [] in the header

Ah, right! So the alternative fix would just be to remove those I guess :)

**Review by fredrikekre — APPROVED** [review]

File: doc/src/manual/interfaces.md (hunk 1 above)
Comment (mbauman) [reply doc/src/manual/interfaces.md]: Or actually link to it. :)

**Pull request merged**